libcnb-test/src/app.rs

```diff
@@ -1,26 +1,14 @@
 use fs_extra::dir::CopyOptions;
```

Comment:
More of a meta comment. I would love some more details in your commits. You break down your actions into very atomic and usually well thought out bits for individual commits (great). I sometimes don't know why (not as great). A sentence or two explaining what was happening before and how it changed would be stellar. 

In this case, the subject line on the commit says what you did. But I'm not totally sure why.

Reply:
In general, I don't think PRs in this repo have been structured with the intention of reviewing by commit (at least not for net new feature work). Many of the commits in this PR are just a side-effect of review comments or refactoring, rather than meant for human consumption.

Reply:
For a new-new feature like this, I don't think it's viable to have nice step-by-step commits. During development, the design changed dramatically between iterations. We would've ended with at least 150 commits that didn't effect the end-result at all. Plus curating every step with long descriptions does not seem to be a good time investment to me.

After I was confident that the design will not change dramatically anymore, I switched gears and kept separate commits that address individual PR comments.